dashboard/app/controllers/projects_controller.rb

```diff
@@ -300,7 +300,7 @@ def show
     # for sharing pages, the app will display the footer inside the playspace instead
     no_footer = sharing
     # if the game doesn't own the sharing footer, treat it as a legacy share
-    @is_legacy_share = sharing && !@game.owns_footer_for_share?
+    @legacy_share_style = sharing && !@game.owns_footer_for_share?
```

Comment:
In the projects controller, we only ever set `legacy_share_style` (in some cases)